api/v1alpha1/jobset_webhook_test.go

```diff
@@ -64,6 +68,37 @@ func TestJobSetDefaulting(t *testing.T) {
 									CompletionMode: completionModePtr(batchv1.NonIndexedCompletion),
 								},
 							},
+							Network: &Network{EnableDNSHostnames: pointer.Bool(true)},
+						},
+					},
+				},
+			},
+		},
+		{
+			name: "enableDNSHostnames is unset",
```

Comment:
do we have a test that sets it explicitly to false?

Reply:
added